.github/workflows/ci.yml

```diff
@@ -87,3 +87,20 @@ jobs:
       with:
         user: meganrm
         password: ${{ secrets.PYPI_TOKEN }}
+
+  cleanup:
```

Comment:
this workflow runs on every pull request, so wouldn't it create files and then immediately delete them? I think we want a separate workflow that has this job that is triggered by a pull request being merged. Have you tested this?

Reply:
I also looked at the output of this and $BRANCH_NAME is being interpreted as a string. You can set env variables directly, you don't need to run `echo "BRANCH_NAME=${{ github.ref }}" > $GITHUB_ENV`
https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
If you forked this repo, you could be opening and closing your own PRs to make sure this got triggered correctly. Often when I'm doing GitHub actions I first try stuff out in my own dummy repo so I don't have to worry about reverting any changes.

Reply:
Also, once you fix the error in the main workflow, this'll run on this PR so you'll be able to see what's happening

Reply:
https://github.com/mesoscope/cellpack/actions/runs/4547324139